### DISCUSSION TITLE
Open Source License?

### DISCUSSION THREAD

**Discussion by swyxio** [breadthe/sd-buddy#20] [General]
something i forgot to discuss is the licensing of this. there's [this](https://github.com/breadthe/sd-buddy#license) in the readme, which means this is source available. i raised the formal licensing question [with the other GUI project too](https://github.com/divamgupta/diffusionbee-stable-diffusion-ui/issues/5) but i dont really like their approach of just copying the SD license. personally prefer MIT or GPL, something OSI recognized.

no pressure of course, its your project

**Comment by breadthe** [comment]
Thanks for bringing this up. I definitely need to update it at some point. MIT or GPL sounds right... probably.

The main idea is anybody can use the code freely, but they can't profit from it.
